src/components/v2/shared/FundingCycleConfigurationDrawers/TokenDrawer.tsx

```diff
@@ -30,8 +31,16 @@ export default function TokenDrawer({
         onClose={handleDrawerCloseClick}
       >
         <p>
-          <Trans>Configure your project's tokenomics.</Trans>
+          <Trans>Design how your tokens should work.</Trans>
         </p>
+        <Callout style={{ marginBottom: '2rem' }}>
+          <Trans>
+            Project tokens <strong>aren't ERC-20 tokens</strong> by default.
```

Comment:
I think maybe you just want to bold *aren't* . This way it stands out more to users

Reply:
Hmm, i dont feel strongly but i kinda wanted to emphasize **ERC-20 tokens** too, because I thought that's what users might be looking for/will resonate more strongly.

Give you've approved I'll merge, but let's def follow up if it doesn't feel right.